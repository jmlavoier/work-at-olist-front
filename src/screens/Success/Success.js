import React from 'react';
import Box from 'components/Box';

import IconRight from 'components/IconRight';

import styles from './Success.sass';

const Success = () => (
  <div className={styles.success}>
    <Box>
      <div className={styles.icon}>
        <IconRight />
      </div>
      <div className={styles.body}>
        <h2>Tudo certo</h2>
        <p>
          Verifique sua caixa de entrada para confirmar seu e-mail.
        </p>
      </div>
    </Box>
  </div>
);

export default Success;
