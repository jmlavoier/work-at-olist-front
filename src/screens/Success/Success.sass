@import 'styles/variables'
@import 'styles/helpers'

.success
  @extend .text-default
  color: $font-color-dark-gray
  text-align: center

  h2
    margin: 20px 0

  p
    max-width: 249px
    margin: 0 auto

.icon
  margin-top: 92px

.body
  margin-bottom: 20px