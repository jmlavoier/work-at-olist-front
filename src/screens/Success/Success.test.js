import React from 'react';
import { shallow } from 'enzyme';

import Success from './Success';

describe('<Success />', () => {
  it('Should component match snapshot', () => {
    const wrapper = shallow(<Success />);

    expect(wrapper).toMatchSnapshot();
  });
});
