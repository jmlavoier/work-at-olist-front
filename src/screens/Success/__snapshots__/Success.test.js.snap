// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Success /> Should component match snapshot 1`] = `
<div
  className="success"
>
  <Box>
    <div
      className="icon"
    >
      <IconRight />
    </div>
    <div
      className="body"
    >
      <h2>
        Tudo certo
      </h2>
      <p>
        Verifique sua caixa de entrada para confirmar seu e-mail.
      </p>
    </div>
  </Box>
</div>
`;
